src/util/lock.ts

```diff
@@ -2,14 +2,14 @@ import { exec, ExecException } from 'child_process';
 
 export const DEFAULT_TIME = 1704844800000; // 2024-01-10, dashcam default time is less than this date. So once the date is bigger, we know that system time is set
 let lockTime = 0;
-let isTimeSet = false;
+let timeSet = false;
 
-export const ifTimeSet = () => {
-  if (isTimeSet) {
+export const isTimeSet = () => {
+  if (timeSet) {
```

Comment:
Is this caching necessary? Seems like overkill to make this a stateful function just to avoid calling Date.now().

Reply:
@mazar-farran that was needed before, no need for this anymore, thank you!